packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.4-dev
```

Comment:
This should either have been 2.0.3-dev.1 of we didn't want to publish, or 2.0.3 if we did (which I think is the case; generally we want to publish README improvements). We've now skipped 2.0.3.

We should update the version bump script to prevent this.